### ISSUE TITLE
Duplicated names when generating functions

### ISSUE THREAD

**Issue by luisggpina** [ccadar/varan-div#28]
The project generates duplicated names for C functions, which then fails to compile, when there are several rules for the same LHS but not defined in sequence.  For instance, the following test (adapted from https://github.com/daniel-grumberg/varan-div/blob/master/test/passing/arg-validation-lhs-override.c) generates a C file that does not compile:

```
open(a, _, _) {{ return ($(a) == 10); }} => write(_,_,_),
                                            self()

read(_,_,_) => gettid(),
               self()

open(_, _, _) => read(_, _, _),
                 getpid(),
                 self()
```

**Comment by daniel-grumberg** [comment]
Getting on it.

**Comment by daniel-grumberg** [comment]
Fixed in #30, I will close this issue when merged

**Comment by luisggpina** [comment]
#30 was merged, I'm closing this issue
